evap/contributor/templates/contributor_index.html

```diff
@@ -95,7 +95,7 @@
                                     {% if course.avg_grade %}
                                         <div class="grade-bg text-center vertically-aligned" style="background-color: {{ course.avg_grade|gradecolor }}" data-toggle="tooltip" data-placement="top" title="{% trans 'Average grade' %}">{{ course.avg_grade|floatformat:1 }}</div>
                                     {% endif %}
-                                    <a href="{% url 'results:course_detail' semester.id course.id %}" class="btn btn-sm btn-secondary" data-toggle="tooltip" data-placement="top" title="{% trans 'Results' %}"><span class="fa fa-bar-chart"></span></a>
+                                    <a href="{% url 'results:course_detail' semester.id course.id %}" class="btn btn-sm btn-secondary btn-results" data-toggle="tooltip" data-placement="top" title="{% trans 'Results' %}"><span class="fa fa-bar-chart"></span></a>
```

Comment:
if there are no results yet, i can sort of click on the disabled results button and the edit page loads, because the whole row is a link to the edit page. if there's an easy fix, please do that, otherwise not.

Reply:
check